src/test/java/seedu/smartnus/logic/commands/EditCommandTest.java

```diff
@@ -277,4 +290,162 @@ public void equals() {
         assertFalse(standardCommand.equals(new EditCommand(INDEX_FIRST_QUESTION, DESC_BOB)));
     }
 
+    @Test
+    public void edit_wrongPanel() {
+        ModelStub model = new EditCommandTest.ModelStub();
+        EditCommand qnCmd = new EditCommand(INDEX_FIRST_QUESTION, DESC_AMY);
+        assertThrows(CommandException.class,
+                Messages.MESSAGE_NOT_IN_QUESTION_PANEL, () -> qnCmd.execute(model));
+    }
+
+    /**
+     * A default model stub that have all of the methods failing.
+     */
+    private class ModelStub implements Model {
```

Comment:
For code quality, perhaps we could abstract this modelstub out into a class that every `test` can use